project-base/app/tests/FrontendApiBundle/Functional/Blog/Article/BlogArticleTest.php

```diff
@@ -150,6 +151,54 @@ public function testGetBlogArticleReturnsErrorWithWrongSlug(): void
         $this->assertSame($expectedErrorMessage, $errors[0]['message']);
     }
 
+    /**
+     * @param string $slug
+     * @param string $expectedImage
+     */
+    #[DataProvider('getImagesDataProvider')]
+    public function testGetBlogArticleImages(string $slug, string $expectedImage): void
+    {
+        $query = '
+            query {
+                blogArticle(urlSlug: "' . $slug . '") {
+                    name
+                    images {
+                        url
+                    }
+                }
+            }
+        ';
+
+        $response = $this->getResponseContentForQuery($query);
+        $responseData = $this->getResponseDataForGraphQlType($response, 'blogArticle');
+
+        $this->assertArrayHasKey('images', $responseData);
+        $this->assertCount(1, $responseData['images']);
+        $this->assertArrayHasKey('url', $responseData['images'][0]);
+        $this->assertStringEndsWith($expectedImage, $responseData['images'][0]['url']);
+    }
+
+    /**
+     * @return iterable
+     */
+    public static function getImagesDataProvider(): iterable
+    {
+        yield [
+            'slug' => 'blog-article-for-search-testing',
+            'expectedImage' => '600.jpg',
+        ];
+
+        yield [
+            'slug' => 'blog-article-for-products-testing',
+            'expectedImage' => '601.jpg',
+        ];
+
+        yield [
+            'slug' => 'grapesjs-page',
```

Comment:
I have the same worry here regarding the failing test after changing the first domain locale...

Reply:
good thinking :) thanks, changed